docs/source/gbd2019_models/risk_risk_correlation/maternal_bmi_hgb_birthweight/index.rst

```diff
@@ -0,0 +1,179 @@
+
+.. math::
+
+   BW_\text{cat4 - GBD} = p_\text{cat1} * BW_\text{cat1 - cat4}
+                           + p_\text{cat2} * BW_\text{cat2 - cat4}
+                           + p_\text{cat3} * BW_\text{cat3 - cat4}
+
+Where :math:`p_\text{cat}` represents the exposure prevalence of a given maternal BMI/hemoglobin category.
+
+Therefore, the association bewteen maternal BMI/hemoglobin joint risk exposure and birthweight risk exposure should be implemented according to the following steps:
+
+#. Assign maternal hemoglobin risk exposure values as described on the :ref:`hemoglobin exposure document <2019_hemoglobin_model>`
+
+#. Assign maternal BMI risk exposure values as described on the :ref:`maternal BMI risk exposure conditional on hemoglobin status document <2019_risk_exposure_maternal_bmi_hgb>`
+
+#. At *conception* (transition from *np* to *p* states of the :ref:`pregnancy model <other_models_pregnancy>`), assign a LBWSG exposure as described on the :ref:`GBD 2019 Low birthweight short gestation risk exposure page <2019_risk_exposure_lbwsg>`
+
+#. Apply a shift to the assigned continuous birthweight exposure value from step 2 based on the assigned maternal BMI exposure such that:
+
+.. math::
+
+   BW_\text{i, shifted} = BW_\text{i, unshifted} + shift_\text{cat_i, adjusted}
+
+Where,
+
+.. list-table:: Adjusted birthweight shifts
+   :header-rows: 1
+
+   *  - Parameter
+      - Adjusted shift
+      - Note
+   *  - :math:`shift_\text{cat4, adjusted}`
+      - :math:`p_\text{cat1} * shift_\text{cat1, unadjusted} + p_\text{cat2} * shift_\text{cat2, unadjusted} + p_\text{cat3} * shift_\text{cat3, unadjusted}`
+      - 
+   *  - :math:`shift_\text{cat3, adjusted}`
+      - :math:`shift_\text{cat4, adjusted} + shift_\text{cat3, unadjusted}`
+      - 
+   *  - :math:`shift_\text{cat2, adjusted}`
+      - :math:`shift_\text{cat4, adjusted} + shift_\text{cat2, unadjusted}`
+      - 
+   *  - :math:`shift_\text{cat1, adjusted}`
+      - :math:`shift_\text{cat4, adjusted} + shift_\text{cat1, unadjusted}`
+      - 
+
+.. note::
+
+   These LBWSG exposure values may be later modified by intervention coverage and/or other factors. Note that a shift in continuous LBWSG exposure values may cause a simulant's going LBWSG exposure values to no longer fall within a valid GBD LBWSG exposure category. However, relative risks for the shifted exposure can still be calculated according ot the :ref:`LBWSG risk effects modeling strategy <2019_risk_effect_lbwsg>`.
+
+   The gestational age assigned to the mother should be used to determine the duration of her pregnancy.
+
+   The gestational age and birtweight exposure values assigned to the mother should be used to determine the child's LBWSG exposure value and relative risks during the neonatal period.
+
+Causation
+++++++++++++
+
+We are not currently modeling a direct causal relationship between changes in maternal BMI exposure and changes in birthweight exposure.
+
+Assumptions and Limitations
```

Comment:
It seems like we assume that the shifts are additive (as opposed to multiplicative, for instance). Is that worth noting here?

Reply:
I added a point to this section regarding how we apply a population mean difference to individuals as an additive shift and the potential implications. Thanks for noting this!